doc/mkdocs/docs/integration/cmake.md

```diff
@@ -65,12 +63,11 @@ target_link_libraries(foo PRIVATE nlohmann_json::nlohmann_json)
 ```
 
 !!! Note
-	The repository <https://github.com/nlohmann/json> download size is huge.
-	It contains all the dataset used for the benchmarks. You might want to depend on
-	a smaller repository. For instance, you might want to replace the URL above by
+	The repository <https://github.com/nlohmann/json> download size quite large.
```

Comment:
Shouldn't it say "The repository ... download size *is* quite large." ?

Reply:
Yes, good catch! Unfortunately, the CI is currently unstable, so it will take some time to fix this.